src/utils/localSearch.ts

```diff
@@ -115,9 +107,14 @@ async function checkResultPermissions(id: string, env: ReleaseEnv = ReleaseEnv.S
   return result;
 }
 
-export const localQuery = async (db: any, term: string, env: ReleaseEnv = ReleaseEnv.STABLE, useGenerated = false) => {
+export const localQuery = async (
+  db: Orama<typeof entrySchema>,
+  term: string,
+  env: ReleaseEnv = ReleaseEnv.STABLE,
+  mode: 'services' | 'quickstarts' = 'services'
```

Comment:
Do we want to limit ourselves only to `services` and `quickstarts`? I think we could provide the search for any mode. We should however provide a pre-build types for `services` and `quickstarts`.